src/index.css

```diff
@@ -46,3 +45,100 @@ code {
     border-radius: 12px;
     padding: 12px;
 }
+h3,h4{
+  text-align: center;
```

Comment:
This is repeated twice in your css file, and besides I feel this should be a helper class rather than targeting the `h3` and `h4` tags. There might be an instance where you might not need to center the tags in a different component.